README.rst

```diff
@@ -21,19 +21,23 @@ Run the test suite
 ------------------
 ``python setup.py test``
 
-Initialize the database
------------------------
-``initialize_bodhi_db``
-
 Migrating Bodhi from v1.0 to v2.0
 ---------------------------------
 ::
 
-    curl -O https://fedorahosted.org/releases/b/o/bodhi/bodhi-pickledb.tar.bz2
-    tar -jxvf bodhi-pickledb.tar.bz2
-    rm bodhi-pickledb.tar.bz2
-    PYTHONPATH=$(pwd) python tools/pickledb.py migrate bodhi-pickledb*
+    curl -O https://infrastructure.fedoraproject.org/infra/db-dumps/bodhi2.dump.xz
+    xzcat bodhi2.dump.xz | psql -U postgres -W
+
+Adjust the development.init file
+--------------------------------
+
+Adjust the configuration key ``sqlalchemy.url`` to point to the postgresql
+database.
+Something like:
+::
+
+    sqlalchemy.url = postgresql://user:password@localhost/bodhi2
```

Comment:
Can you include a section in here too about how to set up postgres with pb_hba.conf, etc?

Should hopefully be able to just copy/paste it from another app of ours.

My only worry is that a new developer won't be able to follow these instructions and will get lost in the postgres setup.  (they need to dnf install the postgres server too, for instance)